src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/CsvFormatResponseIT.java

```diff
@@ -136,6 +139,23 @@ public void arraysAreQuotedInFlatMode() throws IOException {
         setFlatOption(false);
     }
 
+    @Test
+    public void doubleQuotesAreEscapedWithDoubleQuotes() throws IOException {
+
+        final String query = "SELECT * FROM " + TEST_INDEX_NESTED_WITH_QUOTES;
+
+        final CSVResult csvResult = executeCsvRequest(query, false);
+        final List<String> rows = csvResult.getLines();
+        Assert.assertThat(rows.size(), equalTo(2));
+
+        final String expectedValue1 = "\"[{dayOfWeek=6, author=z\"\"z, info=zz}]\"";
```

Comment:
consider inlining variables

Reply:
Again, maybe it's more lines than necessary, but I like explicitly showing what's expected in the tests, and this looks cleaner to me. Not updating, we can discuss more if you disagree.